slack_scrapper.py

```diff
@@ -49,6 +61,8 @@
         textfile.write("ID: {} \n".format(user["id"]))
         textfile.write("------------------------------------------------ \n")
 
+allMessages(sys.args[1])
```

Comment:
At this point, what is going to be the value of `sys.args[1]` seeing as you've already hardcoded the channel to be scrapped for messages.